chainerx_cc/examples/mnist/train_mnist.cc

```diff
@@ -145,7 +136,7 @@ void Run(int64_t epochs, int64_t batch_size, int64_t n_hidden, int64_t n_layers,
                 chx::Array x = test_x.At(indices);
                 chx::Array t = test_t.At(indices);
                 chx::Array y = model(x);
-                loss += SoftmaxCrossEntropy(y, t, false);
+                loss += chx::SoftmaxCrossEntropy(y, t);
```

Comment:
Ditto regarding summation? Also, you might need to include `routines/loss.h`?

Reply:
Fixed!